main.tex

```diff
@@ -180,6 +181,117 @@ \section*{Introduction}
+%
+To evaluate the linkage disequilibrium (LD) of non-synonymous \emph{Vgsc} mutations with the two most widespread \emph{kdr} resistance mutations (\texttt{L995S/F}), the D1 statistic was calculated using haplotypes and the python library Scikit-allel \cite{Miles2016}.
+%%
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsection*{Haplogroup networks}
+
+%
+Discerning the relationships between similar haplotypes can be difficult when using bifurcating trees as, inherently, the distance between the leaves at the tips (haplotypes) will be small.
+%
+As these relationships may be informative of the history of selection, we utilised a network approach to elucidate them.
+%
+Due to large genetic differences between haplogroups, the \texttt{L995S} and \texttt{L995F} bearing haplotypes were analysed separately; haplotypes (variants from 2L:2358158-2431617) were networked using the median-joining algorithm, designed to be used specifically with interspecific data with small genetic distances \cite{Bandelt1999}, implemented in a custom Python script.
+%
+A maximum edge (genetic) distance of two variants was employed and all networks composed of more than two haplotypes are shown.
+%
+Networks were visualised with the Python library graphviz with non-synonymous edges highlighted using the SnpEff “missense” annotations v4.1b \cite{Cingolani2012} and composite figures constructed using Inkscape.
+%%
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsection*{Haplogroup age}
+
+%
+Haplotype age. @@TODO - AM
+-Length of shared haplotype and number of mutations between them are informative of age…
+-Pairwise t values were hierarchically clustered and visualised as a dendrogram using the Python library Scipy and its cluster hierarchy functions linkage method.
+-Cutting the dendrogram at @@generations clustered haplotypes together into haplogroups…
+- Naming of haplogroups with reference to Ag1000g...
+-dendro figure/distro figures/map - Python libraries...
+%%
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsection*{Recombination}
+
+%
+Recombination. @@TODO - AM - Absolute divergence dxy...
+%%
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsection*{Code}
+
+%
+All custom Python code used to run analyses and generate figures is available from \url{https://github.com/malariagen/agam-vgsc-report}. 
+%%
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsection*{Data}
+
+%
+All data, genomic, haplotypic and meta, is publically downloadable via ftp from \url{https://www.malariagen.net}. 
+%%
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section*{Results}
```

Comment:
Can we please please put Methods after Results?

Reply:
Actually, Methods after Discussion. So we go Intro, Results, Discussion, Methods, References.